ipfs-cluster-ctl/main.go

```diff
@@ -79,7 +79,7 @@ func main() {
 		cli.StringFlag{
 			Name:  "host, l",
 			Value: defaultHost,
-			Usage: "host:port of the IPFS Cluster service API",
+			Usage: "host:port of the IPFS Cluster service API\n\t\t",
```

Comment:
Im not sure I understand why you add a line-break followed by 2 tabs? 

I'm fine with simply shortening the description to `host:port of the ipfs-cluster api`.